starter-code/api/api.md

```diff
@@ -122,6 +165,7 @@ Submits a user with the specified data.
     "full_name": $String, //Required. 
     "given_name": $String, //Required.
     "family_name": $String, //Required.
+    "id": $UUID, //Required.
```

Comment:
does Google OAuth return id as a UUID by default? Or is it technically an integer/double. When I inserted this value to MongoDB it automatically set it as a Double

Reply:
Google's id is technically an integer but I assume its always unique